### ISSUE TITLE
[GraphQl] Associations not working with empty graphql configuration

### ISSUE THREAD

**Issue by MartkCz** [api-platform/core#3527]
**API Platform version(s) affected**: https://github.com/api-platform/core/commit/e1e18d496452b6bf7c2d571c3d07a1337daa0409

With following configuration everything working:
```yaml
resources:
    App\Entity\StatusHashTag:
        graphql:
            delete: ~
```

![image](https://user-images.githubusercontent.com/10145362/80277992-1e322880-86f3-11ea-8f6a-16b3a9896f1a.png)

I don't want any operation with `App\Entity\StatusHashTag`, but following configuration:
```yaml
resources:
    App\Entity\StatusHashTag:
        graphql: []
```

now hashTags are missing

![image](https://user-images.githubusercontent.com/10145362/80278025-66514b00-86f3-11ea-82ec-e79bc0bed053.png)

**Comment by alanpoulain** [comment]
I don't understand, if there is no operation, why the resource would be in the schema?

**Comment by MartkCz** [comment]
We don't use StatusHashTag directly (this is reason why graphql is empty array), we use it only as ManyToOne association.

**Comment by MartkCz** [comment]
Duplication: https://github.com/api-platform/core/issues/3136

PR: https://github.com/api-platform/core/pull/3138

Your solution `if (null === $resourceMetadata->getGraphql()) {` is working

**Comment by alanpoulain** [comment]
Will be fixed in 2.5.6.
